de.dlr.sc.overtarget.language.tests/src/de/dlr/sc/overtarget/language/util/TargetFileHandlerTest.xtend

```diff
@@ -0,0 +1,118 @@
+/*******************************************************************************
+ * Copyright (c) 2020 German Aerospace Center (DLR), Simulation and Software Technology, Germany.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0.
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *******************************************************************************/
+package de.dlr.sc.overtarget.language.util
+
+import org.junit.Test
+import org.eclipse.core.resources.ResourcesPlugin
+import org.eclipse.core.resources.IResource
+import java.io.ByteArrayInputStream
+import org.junit.Assert
+import org.junit.Rule
+import org.junit.rules.TemporaryFolder
+import de.dlr.sc.overtarget.language.targetmodel.TargetFile
+import java.util.ArrayList
+
+class TargetFileHandlerTest {
+	
+	val targetFileHandler = new TargetFileHandler	
+	
+	@Rule 
+	public TemporaryFolder tempFolder = new TemporaryFolder()
+	
+	@Test
+	def void testGetTargetFile() {
+		val workspace = ResourcesPlugin.getWorkspace();
+		val root = workspace.getRoot();
+		val projectName = "testProject"
+		val projectWithTarget = root.getProject(projectName);
+		val folder = projectWithTarget.getFolder("target");
+		val tmodelFile = folder.getFile("target.tmodel");
+		projectWithTarget.create(null);
+		if (!projectWithTarget.isOpen()) { 
+			projectWithTarget.open(null)
+		}
+		folder.create(IResource.NONE, true, null);
+
+		val bytes = "
+			Target target {
+				
+			}".getBytes();
+		val source = new ByteArrayInputStream(bytes);
+		tmodelFile.create(source, IResource.NONE, null);
+	
+		val files = new ArrayList
+		val file = targetFileHandler.getTargetFile(tmodelFile, null)
+		val file2 = tempFolder.newFile
+		files.add(file)
+		files.add(file2)
+		for (f : files) {
+			if (f instanceof TargetFile) {
+				Assert.assertTrue("Method found targetFile", true)
```

Comment:
I don' quite get these asserts.... You assert for true and pass it a constant true? And why is it in a if clause?